python/ccxt/test/test_async.py

```diff
@@ -129,7 +129,7 @@ async def call_method(testFiles, methodName, exchange, skippedProperties, args):
 
 
 def exception_message(exc):
-    return '[' + type(exc).__name__ + '] ' + str(exc)[0:500]
+    return '[' + type(exc).__name__ + '] ' + "".join(format_exception(exc, limit=6)
```

Comment:
this is a proper and quite much better log, instead of only exception-message  it shows line number, stacktrace etc..  (limit depth 6 is quite enough. atm with `str` depth was just 1 and even excuded even file/line number)